frontend/packages/dev-console/integration-tests/support/step-definitions/addFlow/export-application.ts

```diff
@@ -0,0 +1,87 @@
+import { When, Then, Given } from 'cypress-cucumber-preprocessor/steps';
+import { devNavigationMenu } from '@console/dev-console/integration-tests/support/constants/global';
+import { createGitWorkload } from '@console/dev-console/integration-tests/support/pages';
+import {
+  navigateTo,
+  projectNameSpace,
+} from '@console/dev-console/integration-tests/support/pages/app';
+import { exportApplication, exportModalButton } from '../../pageObjects/export-application-po';
+import { topologyHelper } from '../../pages/topology-helper-page';
+
+Given('user has created or selected namespace {string}', (componentName: string) => {
+  projectNameSpace.selectOrCreateProject(componentName);
+});
+
+Given('user has created a deployment workload {string}', (workloadName: string) => {
+  navigateTo(devNavigationMenu.Add);
+  createGitWorkload(
+    'https://github.com/sclorg/nodejs-ex.git',
+    workloadName,
+    'Deployment',
+    'nodejs-ex-git-app',
+  );
+});
+
+Given('user is at Add page', () => {
+  navigateTo(devNavigationMenu.Add);
+});
+
+When('user navigates to Add page', () => {
+  navigateTo(devNavigationMenu.Add);
+});
+
+When('user navigates to Topology page', () => {
+  navigateTo(devNavigationMenu.Topology);
+});
```

Comment:
why the user navigates to topology?

Reply:
This step is added to verify that the primer export resource shows up in the topology